data/xml/Q17.xml

```diff
@@ -216,102 +232,124 @@
     <year>2017</year>
-    <abstract>Variation in language is ubiquitous, particularly in newer forms of writing such as social media. Fortunately, variation is not random; it is often linked to social properties of the author. In this paper, we show how to exploit social networks to make sentiment analysis more robust to social language variation. The key idea is <i>linguistic homophily</i>: the tendency of socially linked individuals to use language in similar ways. We formalize this idea in a novel attention-based neural network architecture, in which attention is divided among several basis models, depending on the author’s position in the social network. This has the effect of smoothing the classification function across the social network, and makes it possible to induce personalized classifiers even for authors for whom there is no labeled data or demographic metadata. This model significantly improves the accuracies of sentiment analysis on Twitter and review data.</abstract>
-    <href>https://transacl.org/ojs/index.php/tacl/article/download/1024/245</href>
+    <month>December</month>
+    <doi>10.1162/tacl_a_00062</doi>
+    <abstract>Variation in language is ubiquitous, particularly in newer forms of writing such as social media. Fortunately, variation is not random; it is often linked to social properties of the author. In this paper, we show how to exploit social networks to make sentiment analysis more robust to social language variation. The key idea is linguistic homophily: the tendency of socially linked individuals to use language in similar ways. We formalize this idea in a novel attention-based neural network architecture, in which attention is divided among several basis models, depending on the author’s position in the social network. This has the effect of smoothing the classification function across the social network, and makes it possible to induce personalized classifiers even for authors for whom there is no labeled data or demographic metadata. This model significantly improves the accuracies of sentiment analysis on Twitter and on review data.</abstract>
     <pages>295–307</pages>
-    <url>http://www.aclweb.org/anthology/Q17-1021</url>
-    <video href="https://vimeo.com/234952612" tag="video"/>
+    <url>https://www.aclweb.org/anthology/Q17-1021</url>
+    <video href="https://vimeo.com/234952612" tag="video" />
   </paper>
 
   <paper id="1022">
-    <title>Semantic Specialization of Distributional Word Vector Spaces using Monolingual and Cross-Lingual Constraints</title>
+    <title>Semantic Specialization of Distributional Word Vector Spaces using
+                    Monolingual and Cross-Lingual Constraints</title>
     <author><first>Nikola</first><last>Mrkšić</last></author>
     <author><first>Ivan</first><last>Vulić</last></author>
-    <author><first>Diarmuid</first><last>Ó Séaghdha</last></author>
+    <author><first>Diarmuid
+                            Ó</first><last>Séaghdha</last></author>
```

Comment:
Funky spacing here, also possibly misparsing the name?

Reply:
The parsing is from them. 

```xml
<given-names>Diarmuid Ó</given-names> <surname>Séaghdha</surname>
```